system-contract-dapp-playground/README.md

```diff
@@ -28,16 +28,78 @@ cd ./hedera-smart-contracts/system-contract-dapp-playground
 npm install
 ```
 
-## Running the project
+## Initial account and wallet setup (important)
 
-### Running project in `dev` mode
+**_Important_**: Before proceeding, ensure your `Metamask wallet` points to the correct `Hedera network` (testnet or previewnet) and is associated with a valid `Hedera account` (auto-created or lazy-created). This is crucial for the project's proper functioning. If you haven't configured these settings yet, please follow the guidelines below.
+
+**_note_**: The information shared below is specifically targeted at individuals who do not currently hold a `Hedera Portal` account and/or have not connected the `Hedera network` to their `Metamask` wallet.
+
+#### 1. Create Hedera Portal Profile
+
+- Visit the [Hedera Portal](https://portal.hedera.com/register), follow the instructions to create a **_Hedera Testnet_** account.
+
+- Once you have completed the instructions, you will receive a Hedera Testnet account ID (0.0.x) and your private/public key pair on your testnet page. You will need to save the **_HEX Encoded Private Key_** as later, we will be using it to import the account into your `Metamask` wallet.
+
+<img width="509" alt="image" src="./public/assets/docs/Hedera_portal_profile.png">
+
+**_Important_**: While Hedera supports both ECDSA and ED25519 accounts, please use ECDSA since Ethereum only supports ECDSA.
+
+#### 2. Wallet setup (Metamask)
+
+- Inside your `Metamask` wallet, navigate to Settings > Networks > Add network > Add a network manually. A form will appear, where you can input the required network details. It should look similar to the following:
+
+<img width="509" alt="image" src="./public/assets/docs/Metamask_form.png">
+
+- Please use the provided information below to complete the form and add the `testnet` network.
+
+| Field              | Data                                  |
+| ------------------ | ------------------------------------- |
+| Network name       | Hedera Testnet                        |
+| New RPC URL        | https://testnet.hashio.io/api         |
+| Chain ID           | 296                                   |
+| Currency symbol    | HBAR                                  |
+| Block explorer URL | https://hashscan.io/testnet/dashboard |
+
+- Hit `Save` to save the network to your `Metamask` wallet.
+
+- Repeat the same process to add the `previewnet` network using the information below.
+
+| Field              | Data                                     |
```

Comment:
Add 2 references for localnode and then for mainnet.
Local node urls are here https://github.com/hashgraph/hedera-local-node#exposed-endpoints and the chainID for localnode can be found here https://github.com/hashgraph/hedera-json-rpc-relay#configuration